app/forms/network-interface-edit.tsx

```diff
@@ -65,6 +112,76 @@ export function EditNetworkInterfaceForm({
     >
       <NameField name="name" control={form.control} />
       <DescriptionField name="description" control={form.control} />
+      <FormDivider />
+
+      <div className="flex flex-col gap-3">
+        {/* We have to blow this up instead of using TextField to get better text styling on the label */}
+        <div className="mt-2">
+          <label id="transitIps-label" htmlFor="transitIp" className="text-sans-lg">
+            Transit IPs
+          </label>
+          <TextInputHint id="transitIps-help-text" className="mb-2">
+            Enter an IPv4 or IPv6 address, or{' '}
+            <a
+              href={links.transitIpsDocs}
+              className="inline-flex gap-1 children:last:border-0"
+              target="_blank"
+              rel="noreferrer"
+            >
+              learn more about Transit IPs
```

Comment:
If the whole sentence is the link, I would lowercase transit. Capital L is to go with the other suggestion.

```suggestion
              Learn more about transit IPs
```